backend/app/db/hospital_manager.py

```
@@ -0,0 +1,930 @@
+        """
+        ID로 병원 정보 조회
+
+        Args:
+            hospital_id: MongoDB ObjectId (문자열)
+
+        Returns:
+            병원 정보 또는 None
+        """
+        try:
+            result = await self.db.hospitals.find_one({"_id": ObjectId(hospital_id)})
+            return result
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)